src/screens/Notifications/NotificationScreen.tsx

```diff
@@ -0,0 +1,156 @@
+import { CompositeNavigationProp } from '@react-navigation/native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { Component } from 'react';
+import { View, Text, StyleSheet, Alert, TouchableOpacity, FlatList } from 'react-native';
+
+import { images } from 'app/assets';
+import { Header, ScreenTemplate, Button, FlatButton, ButtonType, Image } from 'app/components';
+import { Route, MainCardStackNavigatorParams, RootStackParams, Wallet } from 'app/consts';
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../loc');
+
+interface Props {
+  navigation: CompositeNavigationProp<
+    StackNavigationProp<RootStackParams, Route.MainCardStackNavigator>,
+    StackNavigationProp<MainCardStackNavigatorParams, Route.Notifications>
+  >;
+  wallets: any;
+}
+export class NotificationScreen extends Component<Props> {
+  onChangeEmailPress = () => {
+    Alert.alert('dsadasd');
+  };
+  onDeletePress = () => {};
+
+  onAddEmailPress = () => {
+    this.props.navigation.navigate(Route.AddEmail);
+  };
+
+  renderItem = (item: any) => {
+    return (
+      <TouchableOpacity style={styles.itemRow} onPress={() => this.goToWalletDetails(item.walletId)}>
+        <View style={styles.row}>
+          <Text style={styles.walletName}>{item.name}</Text>
+          <Image source={images.backArrow} style={styles.arrow} resizeMode="contain" />
+        </View>
+        <Text style={styles.caption}>{item.description}</Text>
+      </TouchableOpacity>
+    );
+  };
+
+  goToWalletDetails = (walletId: string) => {
+    this.props.navigation.navigate(Route.WalletDetails, { id: walletId });
+  };
+
+  renderFooter = () =>
+    this.props.wallets ? (
+      <>
+        <Button title={i18n.notifications.change} onPress={this.onChangeEmailPress} />
+        <FlatButton
+          containerStyle={styles.deleteButton}
+          buttonType={ButtonType.Warning}
+          title={i18n.notifications.delete}
+          onPress={this.onDeletePress}
+        />
+      </>
+    ) : (
+      <Button title={i18n.notifications.addEmail} onPress={this.onAddEmailPress} />
+    );
+
+  render() {
+    return (
+      <ScreenTemplate
+        header={<Header isBackArrow={true} title={i18n.settings.notifications} />}
+        noScroll
+        footer={this.renderFooter()}
+      >
+        {this.props.wallets ? (
+          <>
+            <Text style={styles.title}>{i18n.notifications.title}</Text>
+            <Text style={styles.description}>{i18n.notifications.description}</Text>
+            <View style={styles.amountInput}>
+              <Text style={styles.amount}>kamil2912@gmail.com</Text>
```

Comment:
I'd change it to something like "hardcoded-email-address@gmail.com" to do not confuse others.